examples/cloudiot/versions.tf

```diff
@@ -18,7 +18,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 4.0"
+      version = "~> 4.1.0"
```

Comment:
Let's make these `4.1` as with `4.1.0` the pessimistic constraint operator will function as >= 4.1.0, < 4.2.0

```suggestion
      version = "~> 4.1"
```